packages/patternfly-4/react-core/src/components/Chip/Chip.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChipButton from './ChipButton';
+import ChipText from './ChipText';
+import ChipWrapper from './ChipWrapper';
+import { TimesCircleIcon } from '@patternfly/react-icons';
+
+const Chip = ({ isOverflowed, children, id, className, ...props }) => {
+  return !isOverflowed ? (
+    <ChipWrapper className={className} {...props}>
+      <ChipText id={id}>{children}</ChipText>
+      <ChipButton aria-label="Remove" id={`remove_${id}`} aria-labelledby={`remove_${id} ${id}`}>
```

Comment:
You are missing `onClick` for remove button. Clicking on chip anywhere should not call the on click function, this function should be called only for `ChipButton`.

Reply:
Thanks! I can't believe I missed it. I will fix it soon as possible.